ngraph/frontend/onnx_import/src/op/dropout.cpp

```diff
@@ -0,0 +1,110 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <memory>
+
+#include "core/null_node.hpp"
+#include "default_opset.hpp"
+#include "exceptions.hpp"
+#include "ngraph/log.hpp"
+#include "ngraph/node.hpp"
+#include "op/dropout.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace
+            {
+                OutputVector build_dropout(const Node& node, bool training_mode)
+                {
+                    CHECK_VALID_NODE(
+                        node, !training_mode, "Training mode is not supported for Dropout op");
+
+                    const auto input_data = node.get_ng_inputs().at(0);
+                    const bool return_mask = node.get_outputs_size() > 1;
+
+                    if (return_mask)
+                    {
+                        const auto mask = std::make_shared<default_opset::Broadcast>(
+                            default_opset::Constant::create(
+                                ngraph::element::boolean, Shape{}, {true}),
```

Comment:
Can we try create ngraph::element:i32 and then apply Convert to bool on mask node?

Reply:
I've resolved this problem in CPU plugin (friendly name was not passed correctly)

Reply:
Nice:)